openprescribing/openprescribing/settings/base.py

```diff
@@ -151,7 +151,8 @@
                 'frontend.context_processors.support_email',
                 'frontend.context_processors.google_tracking_id',
                 'frontend.context_processors.api_host',
-                'frontend.context_processors.debug'
+                'frontend.context_processors.debug',
+                'pipeline.context_processors.import_in_progress'
```

Comment:
Add a trailing comma?

Reply:
Haha! You know I actually did out of habit and then realised that the list didn't have one previously and so removed it for consistency...

Reply:
I'm back on another branch now so I think that might be an enhancement for another day

Reply:
👍